modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/value/SdkValueAdapter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.value;
+
+import org.mule.runtime.api.value.Value;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Adapter of the legacy {@link Value} to {@link org.mule.sdk.api.values.Value}
+ *
+ * @since 4.4.0
+ */
+public class SdkValueAdapter implements Value {
```

Comment:
This should be the other way around. Adapt the extension-api verion of it into an sdk-api version as any new features will be added on the `sdk-api` one. This class should actually be a  `LegacyValueAdapter`